### PR TITLE
added update install location

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2182

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-provision.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The remainder of the provisioning includes Windows Hello for Business requesting
 > Read [Azure AD Connect sync: Scheduler](https://docs.microsoft.com/azure/active-directory/connect/active-directory-aadconnectsync-feature-scheduler) to view and adjust the **synchronization cycle** for your organization.
 
 > [!NOTE]
-> Windows Server 2016 update [KB4088889 (14393.2155)](https://support.microsoft.com/help/4088889) provides synchronous certificate enrollment during hybrid certificate trust provisioning.  With this update, users no longer need to wait for Azure AD Connect to sync their  public key on-premises.  Users enroll their certificate during provisioning and can use the certificate for sign-in immediately after completeling the provisioning. 
+> Windows Server 2016 update [KB4088889 (14393.2155)](https://support.microsoft.com/help/4088889) provides synchronous certificate enrollment during hybrid certificate trust provisioning.  With this update, users no longer need to wait for Azure AD Connect to sync their  public key on-premises.  Users enroll their certificate during provisioning and can use the certificate for sign-in immediately after completeling the provisioning. The update needs to be installed on the federation servers.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-provision.md]: Copy checked, no changes.

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #2182?

Thank you.

**Comment by mypil** [comment]
@DaniHalfin  - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #2182?

Thank you.

**Comment by DaniHalfin** [comment]
Looks right. @mapalko please confirm.

**Comment by mypil** [comment]
@DaniHalfin @mapalko - can you please assist to check, approve and merge the content for this PR based on issue #2182?

Thank you.

**Comment by mypil** [comment]
@DaniHalfin @mapalko - I would like to follow up on this. can you please assist to check, approve and merge the content for this PR based on issue #2182?

Thank you.

**Comment by mapalko** [comment]
@mypil @DaniHalfin looks correct to me.

**Comment by mypil** [comment]
@DaniHalfin - Kindly assist to merge this PR since @mapalko confirmed that everything looks correct. Thank you.

**Comment by mypil** [comment]
@DaniHalfin - I would like to follow up on this. Kindly assist to merge this PR since @mapalko confirmed that everything looks correct. 

@AndreaBarr @kenwith - would you be able to merge the content of this PR? 

Thank you.

**Pull request merged**